src/main/java/com/revature/services/QuestionLogicImpl.java

```diff
@@ -40,4 +59,42 @@
 
 	}
 	
+	@Override
+	public QuestionPool getQuestionById(Integer id) {
+		return dao.findOne(id);
+	}
+	
+	@Override
+	public QuestionComment getCommentById(Integer id) {
+		return commentDao.findOne(id);
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1135)